admin/src/server.js

```diff
@@ -33,7 +31,8 @@ function runSubprocess(cmd: string, argv: Array<string>, env: Object, uid: ?numb
       opts.setgid = gid
     }
    
-    return new Promise((resolve, reject) => {
+    return new Promise((resolve) => {
+      /* eslint-disable-next-line camelcase */
       child_process.spawn(cmd, argv, opts)
```

Comment:
Let's just change this to `childProcess`.

Reply:
Agreed -- we should just rename the import.

Reply:
Fixed